website/src/components/subscriptions/create/TriggerTypeInput.tsx

```diff
@@ -0,0 +1,118 @@
+        type: 'count',
+        count: 0,
+    });
+
+    const [countTrigger, setCountTrigger] = useState({
+        type: 'count' as const,
+        count: 0,
+    });
+    const [proportionTrigger, setProportionTrigger] = useState({
+        type: 'proportion' as const,
+        proportion: 1.0,
+    });
+
+    const updateCountTrigger = (newCountTrigger: { count: number; type: 'count' }) => {
+        setCountTrigger(newCountTrigger);
+        setTriggerType(newCountTrigger);
+    };
+
+    const updateProportionTrigger = (newProportionTrigger: { proportion: number; type: 'proportion' }) => {
+        setProportionTrigger(newProportionTrigger);
+        setTriggerType(newProportionTrigger);
+    };
+
+    useEffect(() => {
+        onTriggerTypeChange(triggerType);
+    }, [triggerType]);
+
+    return (
+        <InputLabel
+            title={'Threshold'}
+            description={
+                'The threshold for the subscription. ' +
+                `If the ${triggerType.type === 'count' ? 'count' : 'prevalence'} of the variant exceeds this threshold, ` +
+                'you will receive a notification.'
+            }
+        >
+            <div className='flex w-full'>
+                <label className='w-full'>
+                    <select
+                        className='select select-bordered w-full'
+                        onChange={(event) => {
+                            const value = event.currentTarget.value;
+                            if (value === 'count') {
+                                setTriggerType(countTrigger);
+                            }
+                            if (value === 'proportion') {
+                                setTriggerType(proportionTrigger);
+                            }
+                        }}
+                        value={triggerType.type}
+                    >
+                        <option value={'proportion'}>Prevalence</option>
+                        <option value={'count'}>Count</option>
+                    </select>
+                </label>
+                {triggerType.type === 'count' && (
+                    <label className={`input input-bordered flex items-center gap-2`}>
+                        <input
+                            type='number'
+                            step={1}
+                            min={0}
+                            lang='en'
+                            className={`w-32 grow`}
+                            value={countTrigger.count}
+                            onChange={(event) => {
+                                const newCountTrigger = {
+                                    ...countTrigger,
+                                    count: parseInt(event.currentTarget.value, 10),
+                                };
+                                updateCountTrigger(newCountTrigger);
+                            }}
+                        />
+                    </label>
```

Comment:
If I type letters in those number input field, then the value simply disappears. It's probably even better to store string values and then validate later whether it's a valid number?
But we can also do that in a later PR.

Reply:
There is #148, in which the fields should be validated.